show-land/sass/_base.scss

```diff
@@ -0,0 +1,7 @@
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+}
```

Comment:
We have defined shared directory to re-use these kinds of reset codes from there, you don't need to add it here, you can re-use from that folder